todo_app/data/session_items.py

```diff
@@ -1,69 +1,69 @@
-from flask import session
+# from flask import session
```

Comment:
Similarly here, let's just delete this file - if we need it back we can recover it from git

Reply:
done

Reply:
Doesn't look done. Did you not commit/push the change?